src/printer/cvc/cvc_printer.cpp

```diff
@@ -373,8 +373,10 @@ void CvcPrinter::toStream(
           if( n.getNumChildren()==1 ){
             out << "TUPLE";
           }
-        }else if( t.isRecord() ){
-          const Record& rec = t.getRecord();
+        }
+        else if (t.toType().isRecord())
+        {
+          const Record& rec = static_cast<DatatypeType>(t.toType()).getRecord();
```

Comment:
(see other comments about `static_cast<DatatypeType>`)

```suggestion
          const Record& rec = static_cast<DatatypeType>(t.toType()).getRecord();
```